packages/react-composites/src/composites/CallComposite/types/CallControlOptions.ts

```diff
@@ -1,7 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-/* @conditional-compile-remove-from(stable): custom button injection */
```

Comment:
Are we going to say we wanna check this into stable directly?

Reply:
This is defined unconditionally, but only exported for beta.

We've found that this pattern makes for cleaner code and smaller diff between the build flavors -- define the types unconditionally internally, but only export in the beta flavor.